composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "meilisearch/search-bundle",
+  "name": "meilisearch-local/search-bundle",
```

Comment:
This looks wrong 🤔

Reply:
Github is messing with me today 😠. I'm trying to test how breaking my changes are but it is not easy today!

Reply:
I have a simple symfony app locally and when I try to install this is happening:

```bash
Generating optimized autoload files
62 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  Symfony\Component\ErrorHandler\Error\ClassNotFoundError {#64
!!    #message: """
!!      Attempted to load class "MeiliSearchBundle" from namespace "MeiliSearch\Bundle".\n
!!      Did you forget a "use" statement for another namespace?
!!      """
!!    #code: 0
!!    #file: "./vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php"
!!    #line: 94
!!    trace: {
!!      ./vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:94 { …}
!!      ./vendor/symfony/http-kernel/Kernel.php:386 { …}
!!      ./vendor/symfony/http-kernel/Kernel.php:786 { …}
!!      ./vendor/symfony/http-kernel/Kernel.php:128 { …}
!!      ./vendor/symfony/framework-bundle/Console/Application.php:168 { …}
!!      ./vendor/symfony/framework-bundle/Console/Application.php:74 { …}
!!      ./vendor/symfony/console/Application.php:166 { …}
!!      ./bin/console:43 {
!!        › $application = new Application($kernel);
!!        › $application->run($input);
!!        › 
!!        arguments: {
!!          $input: Symfony\Component\Console\Input\ArgvInput {#3 …}
!!        }
!!      }
!!    }
!!  }
!!  2022-12-19T21:26:57+00:00 [critical] Uncaught Error: Class "MeiliSearch\Bundle\MeiliSearchBundle" not found
!!  
Script @auto-scripts was called via post-update-cmd
```

Reply:
Now I can ensure all the files loaded by my app are the same as this branch without caching issues.

Reply:
Fixed :)